Source/Scene/LabelCollection.js

```diff
@@ -599,7 +599,7 @@ define([
      * calling <code>update</code>.
      *
      * @exception {DeveloperError} This object was destroyed, i.e., destroy() was called.
-     *
+     * @memberOf {LabelCollection.prototype}
```

Comment:
These shouldn't be needed.  We don't do this for our prototype functions in any other class

Reply:
I have seen it in various places of the code https://github.com/AnalyticalGraphicsInc/cesium/search?p=4&q=memberOf&type=Code and I thought that tsd-jsdoc was failing due to this one. Is there any similar file such as `LabelCollection` that I could check out instead?

Reply:
Yes, we use `@memberof` for properties added to prototype via `defineProperties` but shouldn't need it for class methods

Reply:
`BillboardCollection` is very similar to `LabelCollection`

Reply:
Thanks for the clarification :smiley: I will update the PR accordingly.